integration_test/setup.sh

```diff
@@ -1,33 +1,22 @@
 #!/bin/bash
 
-. .env
+CURRENT_WORKDIR=$PWD
+COMPUTATION_ENGINE="../computation-engine-at-orchestrator"
+ENV_FILE="${CURRENT_WORKDIR}/.env"
```

Comment:
Another name like `.env.tests` would avoid replacing the existing `.env` file when running locally.

Reply:
Fixed in d0de2316f6858f0cde975d939ec42b49cdfcf206